src/hooks/useFetchRecommendationPod.ts

```diff
@@ -20,22 +21,24 @@ const useFetchRecommendationPod = (
           })
         )
       );
-      const recommendationPodResults = {};
-
+      const recommendationsPodResults = {};
+      const recommendationsPodsData = {};
       responses.forEach(({ response }, index) => {
         const { pod, results } = response;
         if (pod?.id) {
-          recommendationPodResults[pod.id] = results?.map((item: Item) => ({
+          recommendationsPodResults[pod.id] = results?.map((item: Item) => ({
             ...item,
             id: item?.data?.id,
             section: recommendationPods[index]?.indexSectionName,
             podId: pod.id,
```

Comment:
We already did it once and added the podId. Can we also add sectionDisplayName here as well instead of having separate podData?

Reply:
We could but I don't particularly like it mainly because the sectionDisplayName would exist per item rather than at the top level which could get messy. From what I understand, `podId` needs to exist per item for tracking-related things but `displayName` doesn't so I would rather not put it there.

An alternative implementation, and this might attend to Enes' idea, we could modify the `recommendationsPodResults` to be an object rather than an array. That way we can return one object rather than two but this will be change the expected interface.

Reply:
I agree that modifying `recommendationsPodResults` would be the ideal thing here so I thought adding sectionDisplayName would be less intrusive and wouldn't require passing extra parameters.

I'm ok if you want to keep it the way you implemented though

Reply:
We'll opt of merging it before calling `setActiveSectionsWithData`